src/db/models/chunk.js

```diff
@@ -13,6 +13,8 @@ export const CHUNK_UPLOAD_STATUS = {
     ENQUEUED: 'ENQUEUED',
     IN_PROGRESS: 'IN_PROGRESS',
     COMPLETED: 'COMPLETED',
+    VALIDATED: 'VALIDATED',
+    INVALID: 'INVALID',
```

Comment:
Also I don't see a reason to add one more status - we could have "validating" status with some timeout, reaching which should move chunk to "failed" state. In your logic it looks like after the first validation is failed (which is totally expected since mining can take some time), we consider chunk as "invalid" and don't check it anymore.